sched/sched/sched_timerexpiration.c

```diff
@@ -106,6 +106,24 @@ static unsigned int g_timer_interval;
  * Private Functions
  ****************************************************************************/
 
+static inline_function void set_g_time_tick(clock_t tick)
```

Comment:
let's return the old value:
```suggestion
static inline_function clock_t set_timer_tick(clock_t tick)
```

Reply:
done